lib/global_config.js

```diff
@@ -157,3 +159,52 @@ const rest = {
   HTTPS_PROTOCOL: 'https'
 };
 exports.rest = rest;
+
+/**
+ * As a default we set parameters values identical like in fast-xml-parser lib defaults
+ * thus preserving backward compatibility if customer doesn't set custom configuration
+ */
+const defaultXmlParserConfiguration = {
+  ignoreAttributes: true,
+  alwaysCreateTextNode: false,
+  attributeNamePrefix: '@',
+  attributesGroupName: null
+};
+
+let xmlParserConfiguration = null;
+
+/**
+ * Return custom XmlParser configuration or default if not exists.
+ *
+ * @returns {object}
+ */
+exports.getXmlParserConfiguration = function ()
+{
+  if (xmlParserConfiguration === undefined || xmlParserConfiguration  === null) {
+    return defaultXmlParserConfiguration;
+  }
+  return xmlParserConfiguration;
+}
+
+/**
+ * Set custom xmlParser configuration.
+ * Parametrs that you can override:
+ *   ignoreAttributes - default true,
+ *   attributeNamePrefix - default '@',
+ *   attributesGroupName - default null,
+ *   alwaysCreateTextNode - default false
+ *
+ * @param {object} value
+ */
+exports.setXmlParserConfiguration = function (xmlParamers)
+{
+  if (Util.exists(xmlParamers)) {
+    const resultConfig = {
+      ignoreAttributes: xmlParamers && xmlParamers.ignoreAttributes !== undefined && xmlParamers.ignoreAttributes !== null ? xmlParamers.ignoreAttributes : defaultXmlParserConfiguration.ignoreAttributes,
```

Comment:
you know that xmlParamers exists and don't need to check it in each line

Reply:
ok

Reply:
This part of code not exists in current version